public/style.css

```
@@ -31,7 +31,7 @@
 }
 
 body {
-  background-image:url(/public/backGround.jpg);
+  background-image:url(https://ajup86.github.io/browser-quiz-project-starter/public/backGround.jpg);
```

Have you tried this? They are in the same folder 😄 
```suggestion
  background-image:url(backGround.jpg);
```